### PR TITLE
Uninstall all button w/ confirmation and language strings

### PR DESCRIPTION
**Pull request by Bo98**
@ExtReMLapin said he doesn't know HTML and could not add the confirmation. I cherry picked his commit and added the rest in my own commit so everything is under one PR but still giving some credit to him.

The language strings will need to be uploaded to Crowdin for translation and rebuilt before the proper strings will show in-game.

### CODE REVIEW

**Comment by zerfgog** [comment]
:+1:+1

**Comment by Pigbear** [comment]
do this shit already jesus

**Comment by actowolfy** [comment]
:+1:+1

**Comment by DrVrej** [comment]
Mother of Buttons +1

**Comment by xaviergmail** [comment]
+3

**Comment by robotboy655** [comment]
+somebodyWantsToGetBanned

**Comment by LennyPenny** [comment]
+notMeLOL

**Comment by xaviergmail** [comment]
Are you trying to imply that I want to get banned for supporting a PR and a request?

**Comment by robotboy655** [comment]
This is not supporting, this is spam and annoying everybody who has notifications turned on.
Posting +1's is not going to get the PR merged faster.

**Comment by Harold-Admin** [comment]
You are being watched. The government has a secret system: a machine that spies on you every hour of every day. I know, because I built it. I designed the machine to detect acts of terror, but it sees everything. Violent crimes involving ordinary people; people like you. Crimes the government considered 'irrelevant'. They wouldn't act, so I decided I would. But I needed a partner, someone with the skills to intervene. Hunted by the authorities, we work in secret. You'll never find us, but victim or perpetrator, if your number's up... we'll find you

**Comment by JamieH** [comment]
+1

**Comment by Pigbear** [comment]
The idea of it is to cause you to notice it and merge the damn thing like were asking

**Comment by wiox** [comment]
Perhaps it would be merged faster if the PR didn't modify as many unnecessary lines of code.

File: garrysmod/html/js/menu/Subscriptions.js [hunk 1]
```diff
@@ -25,21 +24,32 @@ Subscriptions.prototype.Contains = function ( id )
 //
 // IsEnabled
 //
-Subscriptions.prototype.Enabled = function ( id )
+Subscriptions.prototype.Enabled = function( id )
 {
```
Comment (MrUnknownGamer) [inline garrysmod/html/js/menu/Subscriptions.js]: Ccc

**Comment by willox** [comment]
Not happening due to the (actual) potential for misclicks. A better idea would be some sort of UI that allows bulk operations on subscribed addons.

**Comment by Kefta** [comment]
Just add a confirmation menu?

**Comment by willox** [comment]
It sounds reasonable, but there are people with so many subscriptions that even the chance of pressing through them accidentally isn't exactly great. Ideally we need a list of previous subscriptions to go alongside it.

**Comment by ExtReMLapin** [comment]
Well, afaik there is a confirmation button/menu in this commit, no ?

**Comment by Bo98** [comment]
Yes, the whole point of me taking @ExtReMLapin's pull request and adding a commit of mine was to add a confirmation button since he didn't know how.

> Ideally we need a list of previous subscriptions to go alongside it.

Do you mean to save it as a file or what? The person should already know what's being uninstalled by the wording of the button.

**Comment by DrVrej** [comment]
We really need one asap

**Comment by Bo98** [comment]
#1122

**Pull request closed without merging**